### ISSUE TITLE
Browse/Import/Copy Pre-built decks

### ISSUE THREAD

**Issue by mkaatman** [monovertex/ygorganizer#27]
I didn't see an item open for this yet, but it would be neat if we had a list of all the pre-existing starter/structure decks. 

You could browse those decks, import them into your collection and copy (fork) the deck for modification. The 3x Deck combos seem to be popular starting points. I wonder if it would make sense to have 1x,2x,3x starting points for every deck.

We could probably link to these decks from the list of sources in the card details as well.

**Comment by monovertex** [comment]
I'd say we should simply do this manually through the app, once the deck builder is complete and if we have that deck "cores" feature implemented.

Then we won't need to do this a special feature, as I feel they are not so many and I'd rather not program an algorithm just to generate these.

**Comment by mkaatman** [comment]
Agreed. There aren't too many so it shouldn't be too much manual effort.
